docs/extra/k8s-cheatsheet.md

```diff
@@ -0,0 +1,102 @@
+# Kubernetes Cheatsheet
+
+## Session Manager SSH
+
+When using AWS Session Manager since the CLI is very limited you will benefit from using this alias:
```

Comment:
this -> these

Reply:
Next time do code suggestion please. Check bellow for example.